components/pages/home/questAndCollectionTabs.tsx

```diff
@@ -112,9 +113,24 @@ const QuestAndCollectionTabs: FunctionComponent<
     fetchBoosts();
   }, [address]);
 
+  useEffect(() => {
+    const fetchRelevantBoosts = async () => {
+      if(!completedQuestIds) return
+      const relevantBoosts = boosts.filter(
+        (b) =>
+          !((new Date().getTime() - b.expiry) / MILLISECONDS_PER_WEEK <= 3 &&
+            b.expiry < Date.now()) ||
+          b.quests.some((quest) => completedQuestIds.includes(quest))
```

Comment:
Here I would simply keep
```tsx
b.expiry > Date.now()) ||
          b.quests.some((quest) => completedQuestIds.includes(quest))
```
So that it only returns quests that aren't expired or quests that are completed